graph/coloring/coloring.go

```diff
@@ -274,7 +276,7 @@ func bestMaximumClique(g graph.Undirected, cliques [][]graph.Node) (colors map[i
 		return colorClique(cliques[0])
 	}
 
-	sort.Slice(cliques, func(i, j int) bool { return len(cliques[i]) > len(cliques[j]) })
+	slices.SortFunc(cliques, func(a, b []graph.Node) int { return cmp.Compare(len(b), len(a)) })
```

Comment:
I'm not convinced this this is clearer.

Reply:
Fair enough! Given the benchmark results are a wash and this is up to preference, I agree, so I'll revert it.